### DISCUSSION TITLE
about adding an entry for 'Curator' in the info list?

### DISCUSSION THREAD

**Discussion by mrezauli** [awcodes/filament-curator#339] [Ideas]
There is provision for curator entries in the InfoListPackage, similar to the Spatie Media plugin. So, you can add them.

**Comment by awcodes** [comment]
I'll try to work on it, but you could just use a regular ImageEntry if it's a relationship, unless it's a multiple relationship.

```php
ImageEntry::make('featured_image.path')
```
